### DISCUSSION TITLE
Download Individual Episodes from Season Packs

### DISCUSSION THREAD

**Discussion by adirt4289** [rogerfar/rdt-client#577] [General]
Hi, I'm wondering if it's possible to use Prowlarr + rdtclient to download individual episodes from season packs. I often find that I only want specific episodes from a season, but there aren't enough releases of individual episodes on RD. This means I'd have to download the entire season, which can be wasteful if I only need a few episodes.

Thanks in advance

**Comment by rogerfar** [comment]
Yes, you select which files you want before starting a download.

Select your files at 
```
Available files
These files are available for immediate download from your debrid provider.
It is possible that there are more files in the torrent, which are not shown here.
```

Or just let Radarr handle it.

**Reply by adirt4289** [reply]
i dont understand, where do i select the files?

**Reply by Reasonable-Grape2698** [reply]
Use 'include' field when you add the torrent, it's regex

**Reply by adirt4289** [reply]
> Use 'include' field when you add the torrent, it's regex

i dont add the torrent, sonarr does
